wfl/utils/julia.py

```diff
@@ -1,4 +1,12 @@
 import os
+import subprocess
+import shlex
+from pathlib import Path
 
 def julia_exec_path():
     return os.environ.get("WFL_JULIA_COMMAND", "julia")
+
```

Comment:
I like this in a function, but do you really think it belongs outside of `fit/ace.py`?  It's entirely specific to the fitting code. The julia interpreter I can see being useful for other things, but not the fitting script itself.

Reply:
Sorry - I find the github interface for associating comments with specific source lines confusing (or just broken).

I mean the function that returns the `run_train.py` path - don't you think that should live in `fit/ace.py`, or maybe `fit/ace_utils.py`, or even `fit/utils.py`?

Reply:
yeah, that sounds more reasonable. I moved it to `fit/utils.py`.

Reply:
I'm going to modify it further, making `ace_fit_jl_path` look up its own julia path, and add it to the command, so the calling function can do it in one step.  Just running the tests manually now, to make sure it's still OK.

Reply:
yes, that's how it should be 😄